integrations/mongodb_atlas/src/haystack_integrations/document_stores/mongodb_atlas/document_store.py

```diff
@@ -5,9 +5,10 @@
 import re
 from typing import Any, Dict, List, Optional, Union
 
+import numpy as np
```

Comment:
Don't import the whole `numpy` package, it noticeably slows import time of this package.